_scicomputing/store_overview.md

```diff
@@ -49,7 +49,7 @@ You will need to consider the security needs of your data sets before placing da
 
 This table describes some of the most common security needs for data sets and how or if each storage system implements that security capability.
 
-|  Feature 	                | Secure File	| Fast File | Economy Cloud |  Scratch  	|  OneDrive 	|
```

Comment:
We should keep this line in this table until it's been decomissioned.

Reply:
Added back scratch